client/client-multi/client-c/src/kaa/utilities/kaa_log.h

```diff
@@ -28,7 +28,7 @@
 
 #include "../kaa_error.h"
 #include "../platform/defaults.h"
-
+#include "../platform/stdio.h"
```

Comment:
I am afraid of this may crash the build for the Econais platform. Need to check.